### PR TITLE
Update dashboard FAQ

### PR DESCRIPTION
**Pull request by shhdgit**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Added a new FAQ for TiDB Dashboard

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- Other reference link(s): [tidb-dashboard#902](https://github.com/pingcap/tidb-dashboard/issues/902)

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: dashboard/dashboard-faq.md [hunk 1]
```diff
@@ -53,3 +53,7 @@ QPS 及 Latency 监控依赖于集群中已正常部署 Prometheus 监控实例
 ### 概况页面中 Top SQL 语句、最近慢查询显示 `invalid connection` 错误
 
 可能的原因是你开启了 TiDB 的 `prepared-plan-cache` 功能。作为实验性功能，`prepared-plan-cache` 在某些版本的 TiDB 中存在一些缺陷，开启后可能会导致 TiDB Dashboard（及其他应用）出现该问题。请依据[文档](/tidb-configuration-file.md#prepared-plan-cache)关闭 `prepared-plan-cache` 功能。
+
+### 慢查询页面显示 `unknown field` 错误
+
+如果在集群升级后慢查询页面出现 `unknown field` 错误，这是由于升级后新版本 TiDB Dashboard 字段与用户偏好设置的字段（处于浏览器缓存内）不兼容导致报错。此时可以尝试通过[清理浏览器缓存](https://developer.chrome.com/docs/devtools/storage/localstorage/#deleteall)以清空用户偏好设置。
```
Comment (qiancai) [inline dashboard/dashboard-faq.md:59]:
```suggestion
集群升级后，如果慢查询页面出现 `unknown field` 错误，是由于升级后新版本 TiDB Dashboard 字段与浏览器缓存内的用户偏好设置的字段不兼容导致的。此时可以尝试通过[清理浏览器缓存](https://developer.chrome.com/docs/devtools/storage/localstorage/#deleteall)以清空用户偏好设置。
```

**Review by breeswish — APPROVED** [review]

File: dashboard/dashboard-faq.md [hunk 2]
```diff
@@ -53,3 +53,7 @@ QPS 及 Latency 监控依赖于集群中已正常部署 Prometheus 监控实例
 ### 概况页面中 Top SQL 语句、最近慢查询显示 `invalid connection` 错误
 
 可能的原因是你开启了 TiDB 的 `prepared-plan-cache` 功能。作为实验性功能，`prepared-plan-cache` 在某些版本的 TiDB 中存在一些缺陷，开启后可能会导致 TiDB Dashboard（及其他应用）出现该问题。请依据[文档](/tidb-configuration-file.md#prepared-plan-cache)关闭 `prepared-plan-cache` 功能。
+
+### 慢查询页面显示 `unknown field` 错误
+
+集群升级后，如果慢查询页面出现 `unknown field` 错误，是由于升级后新版本 TiDB Dashboard 字段与浏览器缓存内的用户偏好设置的字段不兼容导致的。此时可以尝试通过[清理浏览器缓存](https://developer.chrome.com/docs/devtools/storage/localstorage/#deleteall)以清空用户偏好设置。
```
Comment (breeswish) [inline dashboard/dashboard-faq.md:59]: 感觉我们最好备注一下这个是 bug，在 xxx 版本已修复，可以使用新版本，或在老版本里使用 xxx 方式绕过

Comment (shhdgit) [reply dashboard/dashboard-faq.md:59]: 嗯，主要是这个 bug 还没修复，现在改字段名对于 LocalStorage 里的数据需要怎么处理还是 undefined。可以等 [tidb-dashboard#902](https://github.com/pingcap/tidb-dashboard/issues/902) 解决了之后，再来改 FAQ

**Comment by shhdgit** [comment]
Updated, PTAL.

**Comment by TomShawn** [comment]
@shhdgit Please help confirm whether this PR content applies to v5.1. Thanks~

**Comment by shhdgit** [comment]
Yes, v5.1 can also be applied.

File: dashboard/dashboard-faq.md [hunk 3]
```diff
@@ -56,4 +56,14 @@ QPS 及 Latency 监控依赖于集群中已正常部署 Prometheus 监控实例
 
 ### 慢查询页面显示 `unknown field` 错误
 
-集群升级后，如果慢查询页面出现 `unknown field` 错误，是由于升级后新版本 TiDB Dashboard 字段与浏览器缓存内的用户偏好设置的字段不兼容导致的。此时可以尝试通过[清理浏览器缓存](https://developer.chrome.com/docs/devtools/storage/localstorage/#deleteall)以清空用户偏好设置。
+集群升级后，如果慢查询页面出现 `unknown field` 错误，是由于升级后新版本 TiDB Dashboard 字段与浏览器缓存内的用户偏好设置的字段不兼容导致的。该问题已在 v4.0.14/v5.0.3 及之后的版本修复，老版本可通过清空浏览器 Local Storage 解决。
+
+** 清空浏览器 Local Storage: **
+
+1. 点击 **自定义及控制** 按钮，选择 **更多工具 > 开发者工具**。
```
Comment (breeswish) [inline dashboard/dashboard-faq.md:63]: 这个步骤好像有 bug，它是把当前页面的 local storage 清理了，但不是把 dashboard 的清理了

File: dashboard/dashboard-faq.md [hunk 4]
```diff
@@ -56,4 +56,14 @@ QPS 及 Latency 监控依赖于集群中已正常部署 Prometheus 监控实例
 
 ### 慢查询页面显示 `unknown field` 错误
 
-集群升级后，如果慢查询页面出现 `unknown field` 错误，是由于升级后新版本 TiDB Dashboard 字段与浏览器缓存内的用户偏好设置的字段不兼容导致的。此时可以尝试通过[清理浏览器缓存](https://developer.chrome.com/docs/devtools/storage/localstorage/#deleteall)以清空用户偏好设置。
+集群升级后，如果慢查询页面出现 `unknown field` 错误，是由于升级后新版本 TiDB Dashboard 字段与浏览器缓存内的用户偏好设置的字段不兼容导致的。该问题已在 v4.0.14/v5.0.3 及之后的版本修复，老版本可通过清空浏览器 Local Storage 解决。
+
+** 清空浏览器 Local Storage: **
+
```
Comment (breeswish) [inline dashboard/dashboard-faq.md:62]:
```suggestion

```

File: dashboard/dashboard-faq.md (hunk 3 above)
Comment (shhdgit) [reply dashboard/dashboard-faq.md:60]: 2. 选中 **Application** 面板，展开 **Local Storage** 菜单并选中**Dashboard 域名**，点击 **Clear All** 按钮。

改成这样？

Comment (breeswish) [reply dashboard/dashboard-faq.md:61]: 我觉得可行。最好图片也更新一下，是一个真实的 dashboard 域名

另外这个操作必须在打开 dashboard 后才能进行，可以作为步骤写一下。因为如果只是简单地打开了浏览器再打开开发者工具那么是看不到 Dashboard 域名的

Comment (shhdgit) [reply dashboard/dashboard-faq.md:61]: ok

File: dashboard/dashboard-faq.md [hunk 5]
```diff
@@ -58,10 +58,12 @@ QPS 及 Latency 监控依赖于集群中已正常部署 Prometheus 监控实例
 
 集群升级后，如果慢查询页面出现 `unknown field` 错误，是由于升级后新版本 TiDB Dashboard 字段与浏览器缓存内的用户偏好设置的字段不兼容导致的。该问题已在 v4.0.14/v5.0.3 及之后的版本修复，老版本可通过清空浏览器 Local Storage 解决。
 
-1. 点击 **自定义及控制** 按钮，选择 **更多工具 > 开发者工具**。
+1. 打开 Dashboard 页面
```
Comment (breeswish) [inline dashboard/dashboard-faq.md:61]:
```suggestion
1. 打开 TiDB Dashboard 页面
```

**Review by breeswish — APPROVED** [review]

**Comment by breeswish** [comment]
@TomShawn PTAL, thanks!

**Comment by qiancai** [comment]
This PR is closed because the author created a new PR (https://github.com/pingcap/docs-cn/pull/6559) in master to cover the same changes.

**Pull request closed without merging**